torch/_torch_docs.py

```diff
@@ -2603,6 +2603,13 @@ def merge_dicts(*dicts):
 If :attr:`dim` is not given, it defaults to the first dimension found with the
 size 3. Note that this might be unexpected.
 
+.. seealso::
+        :func:`torch.linalg.cross` which is a variant that has a slightly
+        different default behaviour, i.e dim=-1.
```

Comment:
Perhaps just say: "[...] which has a `-1` as the default dimension".

Reply:
Done.